apps/site/lib/site_web/controllers/schedule/timetable_controller.ex

```diff
@@ -75,61 +75,47 @@ defmodule SiteWeb.ScheduleController.TimetableController do
     end
   end
 
-  @spec trip_messages(Routes.Route.t(), 0 | 1) :: %{{String.t(), String.t()} => String.t()}
-  def trip_messages(%Routes.Route{id: "CR-Haverhill"}, 0) do
-    %{
-      {"221"} => "Via Lowell Line",
-      {"221", "place-WR-0067"} => "Via",
-      {"221", "place-WR-0075"} => "Lowell",
-      {"221", "place-WR-0085"} => "Line"
-    }
-  end
+  @doc """
+  Additional text to be included in the timetable.
 
+  We use this for Commuter Rail trips which travel via atypical routes, in
+  order to match the PDF schedules. Each rating, this should be checked
+  against the new PDFs to ensure it's kept up to date.
```

Comment:
Should we create an Asana ticket or some sort of reminder every time the rating changes so we don't forget about this? cc @thecristen